config/database.yml

```diff
@@ -1,8 +1,14 @@
+# If you want to change this file, please keep the changes in your working
+# copy by using
+#
+#     git update-index --skip-worktree config/database.yml
```

Comment:
@enricostano I'm trying to make both you and @d2bit happy. @d2bit wants a local untracked file (achieved with --skip-worktree), you want a complete database.yml (you have it). I want a pretty much empty production section and no user/password in development, I have it.